code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -1,14 +1,103 @@
+import Radium from 'radium';
 import React from 'react';
 import { STAGE_PROGRESS_TYPE } from './types';
 import { saveAnswersAndNavigate } from '../../levels/saveAnswers';
+import color from '../../color';
+
+var style = {
+  overviewContainer: {
+    display: 'table-cell',
+    verticalAlign: 'middle',
+    paddingRight: '10px'
+  },
+  headerContainer: {
+    padding: '5px 8px',
+    backgroundColor: color.lightest_gray,
+    border: `1px solid ${color.lighter_gray}`,
+    borderRadius: '5px'
+  },
+  dot: {
+    puzzle: {
+      display: 'inline-block',
+      width: '24px',
```

Comment:
Anything not in the list from your link will automatically get px. The one that's gotten me a few times is `lineHeight` which does not auto-get px.

idk about shorthand. Could try `5 8` but I bet it doesnt work.